fc/mef.py

```diff
@@ -607,14 +353,11 @@ def get_transform_fxn(data_beads, peaks_mef, mef_channels,
     ----------
     data_beads : FCSData object
         Flow cytometry data, taken from calibration beads.
-    peaks_mef : array
+    mef_values : array
         Known MEF values of the calibration beads' subpopulations, for
         each channel specified in `mef_channels`.
     mef_channels : int, or str, or list of int, or list of str
         Channels for which to generate transformation functions.
```

Comment:
Still think `mef_values` and `mef_channels` can be more intuitively combined into a `dict` where key=channel (int or str) and value = 1D numpy array. See #146.